test/db/formats/web_assembly

```diff
@@ -85,16 +85,16 @@ CMDS=aa;afl
 EXPECT=<<EOF
 0x00000207    1      3 entry0
 0x00000516    1     19 sym.emscripten_stack_init
-0x0000023d    3    209 sym.main
-0x0000032d    1    126 sym.atoi_via_import
-0x000003b9    1     54 sym.static_int_sum
-0x000003f3    3     14 fcn.000003f3
+0x0000023d    4    209 sym.main
+0x0000032d    2    126 sym.atoi_via_import
+0x000003b9    2     54 sym.static_int_sum
+0x000003f1    3     15 sym._start
```

Comment:
<3

Reply:
Yeah, fixed a couple things there, control flow and function sizes should be a bit more wasm like.